exercises/hexadecimal/example.js

```diff
@@ -4,7 +4,7 @@ export default function (hex) {
   this.toDecimal = () => {
     const hexCharacters = [...this.hex];
 
-    for (let i = 0; i < hexCharacters.length; i++) {
+    for (let i = 0; i < hexCharacters.length; i + 1) {
```

Comment:
I think this might have just been a typo, perhaps it's intended to be `i += 1` instead of `i + 1`?  This may have been the cause of the CI timing out (due to infinite loop), cc @tejasbubane

Reply:
Ahh yes! I missed this. Good catch @serixscorpio

Reply:
Nice catch.  That was it!